core/controllers/suggestion.py

```diff
@@ -770,10 +778,21 @@ def get(self, target_type: str, suggestion_type: str) -> None:
                 .get_suggestions_with_editable_explorations(
                     reviewable_suggestions))
         elif suggestion_type == feconf.SUGGESTION_TYPE_ADD_QUESTION:
+            topic_name = self.normalized_request.get('topic_name')
+            topic_skills = None
+            if (
+                    topic_name is not None
+                    and topic_name != constants.TOPIC_SENTINEL_NAME_ALL):
```

Comment:
Ditto, same question ... I think it would be cleaner to have just one representation for "no specific topic specified". Unless these two values mean different things?